src/commands/sbom/validation.ts

```diff
@@ -48,10 +55,23 @@ export const validateSbomFileAgainstSchema = (path: string, ajv: Ajv, debug: boo
       process.stdout.write('File is a valid CycloneDX 1.4 file\n')
     }
 
-    if (isValid14 || isValid15) {
+    if (isValid14 || isValid15 || isValid16) {
       return true
     }
 
+    // show the errors
+    if (!isValid16) {
+      const errors15 = validateFunctionCycloneDx16.errors || []
```

Comment:
I think there is a typo and either should be `errors16` or all errors array should be named `errors` as its already under a `isValid<version>` scope